### PR TITLE
KBI0002: DataLad dataset from Dataverse dataset

### LINKED ISSUE #8: Create DataLad dataset from Dataverse dataset
Origin: email

OP has deposited a dataset on a dataverse instance. Not via `datalad-dataverse` or any DataLad usage whatsoever. OP needs to have this deposit turned into a DataLad dataset to be able to include it in a DataLad superdataset as a dependency.

An exploration of the problem space is at https://github.com/datalad/datalad-dataverse/issues/292

There are many open Qs re a possible framing in the form of a DataLad command. It seems best to first write down a possible implementation using tooling that is available right now.

TODO (not necessarily to be performed in this order)
- [ ] Inform OP/Add reference to this issue at origin
- [x] Clarifying Qs asked or not needed
- [x] Nature of the issue is understood
- [ ] Inform OP about resolution

### PR DESCRIPTION
**Pull request by mih**
Closes #8

### CODE REVIEW

File: kbi/0002/index.rst [hunk 1]
```diff
@@ -0,0 +1,92 @@
+.. index::
+   pair: dataverse; dataset
+
+KBI0002: Represent a Dataverse dataset as a DataLad dataset
+===========================================================
+
+:authors: Michael Hanke <michael.hanke@gmail.com>
+:discussion:
```
Comment (mih) [inline kbi/0002/index.rst:8]:
```suggestion
:discussion: https://github.com/psychoinformatics-de/knowledge-base/pull/9
```

File: kbi/0002/index.rst [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+have such a DataLad dataset, for example, to be able to declare a (versioned)
+data dependency for using DataLad's built-in provenance tracking features.
+
+
+Requirements
+------------
+
+The solution described here requires DataLad (tested with v0.18), and the
+`Dataverse extension package`_ (tested with v1.0). In principle, this can also
+be down with the `pyDataverse package`_ directly, but this has not been tested.
```
Comment (adswa) [inline kbi/0002/index.rst:25]:
```suggestion
be done with the `pyDataverse package`_ directly, but this has not been tested.
```

File: kbi/0002/index.rst [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+KBI0002: Represent a Dataverse dataset as a DataLad dataset
+===========================================================
+
+:authors: Michael Hanke <michael.hanke@gmail.com>
+:discussion:
+:keywords: addurls, datalad-dataverse
+
+The `Dataverse extension package`_ equips DataLad with the ability to deposit
+on Dataverse_ and later retrieve such DataLad datasets from Dataverse again.
+However, if a dataset has been deposited on Dataverse already, there is no
```
Comment (adswa) [inline kbi/0002/index.rst:13]:
```suggestion
The `Dataverse extension package`_ equips DataLad with the ability to deposit DataLad datasets
on Dataverse_ and later retrieve such DataLad datasets from Dataverse again.
However, if a non-DataLad dataset has been deposited on Dataverse already, there is no
```

I guess it doesn't hurt to be explicit

**Review by adswa — APPROVED** [review]
I left a few minor suggestions, but this overall looks good to me

File: kbi/0002/index.rst [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+details). Moreover, the script requires two positional arguments:
+
+1. the base URL of the Dataverse instance that is hosting the dataset
+2. the persistent identifier (PID) of the Dataverse dataset to be queried
+
+The PID can be found in the "Metadata" tab of the dataset's page on Dataverse.
+
+An example script call looks like::
+
+   $ export DATAVERSE_API_TOKEN=99ad0817-....-....-....-279a39346c70
```
[the remark this answers is not in the document]
Comment (mih) [reply kbi/0002/index.rst:52]:
```suggestion
   # needs to be using the `set` command on Windows
   $ export DATAVERSE_API_TOKEN=99ad0817-....-....-....-279a39346c70
```

**Review by mslw — APPROVED** [review]

**Pull request merged**